### PR TITLE
Update/cleanup JPA sample tests

### PR DESCRIPTION
**Pull request by mezarin**
Fixes Issue: https://github.com/OpenLiberty/application-stack/issues/119

### CODE REVIEW

**Comment by ajm01** [comment]
This looks good to me.

**Review by ajm01 — APPROVED** [review]

File: jpa/README.md
Comment (scottkurz) [inline jpa/README.md:217]: @ajm01 did odo team build any doc off this name?  If not maybe just leave it alone?

File: jpa/src/test/java/org/example/app/it/DatabaseIT.java
Comment (scottkurz) [inline jpa/src/test/java/org/example/app/it/DatabaseIT.java:60]: Can we log this `url`.. I just want to show an IT logging sample.. we were using log4j before I assume since MST libs did.   I think using slf4j-log4j12   (so SLF4J with log4j config) would be a fine choice.

File: jpa/src/main/java/org/example/app/PersonResource.java
Comment (scottkurz) [inline jpa/src/main/java/org/example/app/PersonResource.java:136]: Good that you removed `@POST`

File: jpa/README.md
Comment (ajm01) [reply jpa/README.md:217]: I believe they are in process, not finalized as of yet - I can check with them

File: jpa/src/main/java/org/example/app/PersonResource.java
Comment (mezarin) [reply jpa/src/main/java/org/example/app/PersonResource.java:136]: This was not just about removing the @POST. If you remove just the post, then all you have done is to swap one warning for another. The method needed to just be a simple method.

File: jpa/src/test/java/org/example/app/it/DatabaseIT.java
Comment (mezarin) [reply jpa/src/test/java/org/example/app/it/DatabaseIT.java:60]: I can add  the log4j dependency and print the message.

File: jpa/README.md
Comment (mezarin) [reply jpa/README.md:217]: I will let Andy answer your question before doing anything, but let me just mention that the instructions explicitly state that the database name should sampledatabase:
```
- Click on the 'Create Database' button
- Change the value in the Database Name field to 'sampledatabase'
- Change the value in the Database User field to 'sampleuser'
- Change the value in the Database Password field to 'samplepwd'
```
but the sample outputs show: sampledb as the database name.

Comment (ajm01) [reply jpa/README.md:217]: I think it is fine to standardize it here to one name and I can let them know there was an update to line everything up in the doc.

File: jpa/src/test/java/org/example/app/it/DatabaseIT.java
Comment (scottkurz) [reply jpa/src/test/java/org/example/app/it/DatabaseIT.java]: Thanks

File: jpa/src/main/java/org/example/app/PersonResource.java
Comment (scottkurz) [reply jpa/src/main/java/org/example/app/PersonResource.java:136]: Not sure what you mean... you mean you needed to remove the Bean Validation?

File: jpa/pom.xml
Comment (scottkurz) [inline jpa/pom.xml:99]: I think:
```
<scope>test</scope>   
```

File: jpa/pom.xml
Comment (scottkurz) [inline jpa/pom.xml:105]: Do you need the org.glassfish json dependency?

File: jpa/pom.xml
Comment (scottkurz) [inline jpa/pom.xml:118]: You can delete all these JAXB / activation dependencies.    The javax.xml.bind - >javax.activation ones

**Review by scottkurz — APPROVED** [review]
There are some extra deps in here.  I'll mark approved and leave it up to you if you want to merge.

File: jpa/pom.xml
Comment (scottkurz) [inline jpa/pom.xml:71]: Probably should be test-scoped

**Review by scottkurz — APPROVED** [review]

**Pull request merged**
